scripts/helpers/wallet-precalculation.helper.js

```diff
@@ -41,6 +41,21 @@ export const multisigWalletsData = {
 };
 multisigWalletsData.walletConfig.pubkeys = multisigWalletsData.pubkeys;
 
+export const singleMultisigWalletData = {
```

Comment:
It doesn't actually make sense, but a multisig with a single pubkey is good for testing some cases since we just need to init 1 wallet and get 1 signature.
We will not use this for the actual multisig tests because we need to test all cases which include multiple participants and less the the min participants.

Reply:
We need to at least explain this in a comment and which tests are better with 1-out-of-1 multisig wallet